sources/packages/backend/apps/api/src/route-controllers/program-year/models/program-year.dto.ts

```diff
@@ -1,4 +1,4 @@
-export interface ProgramYearDto {
+export interface ProgramYearAPIOutDTO {
```

Comment:
API in/out DTOs are classes on `API` side. only then it will show up in swagger. like 
![image](https://user-images.githubusercontent.com/77353155/208499976-0db5d93f-a0f0-4ae1-b36f-9d750aaad478.png)
https://github.com/bcgov/SIMS/wiki/Development---Standards,-Decisions,-and-Troubleshooting#controller-dto-models

Reply:
ok. Done. Thanks!